sormas-app/app/src/main/res/values/strings_format.xml

```diff
@@ -23,6 +23,10 @@
     <string name="age_with_dob_and_prepend_format"><b>%1$s:</b> %2$s %3$s (%4$s)</string>
     <string name="date_range_format">%1$s - %2$s</string>
     <string name="due_timeago_format">due %1$s</string>
+    <string name="lbds_response_cases">LBDS Sync Response. Cases transferred: %1$d , ignored: %2$d</string>
+    <string name="lbds_response_persons">LBDS Sync Response. Persons transferred: %1$d , ignored: %2$d</string>
+    <string name="lbds_cases_transferred">LBDS Sync. Cases transferred: %1$d</string>
```

Comment:
Better:
<string name="lbds_send_cases">LBDS Sync. Sending cases: %1$d</string>

"transferred" sound like it's already done

Reply:
done